src/test/java/pages/PepperoniPage.java

```diff
@@ -0,0 +1,56 @@
+package pages;
+
+import elements.Button;
+import elements.Text;
+import endpoints.OnlinerEndpoints;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.WaitUtils;
+
+public class PepperoniPage extends BasePage {
+
+    @FindBy(className = "product-aside__price--primary")
+    protected Text price;
+
+    @FindBy(css = ".button-style.button-style_expletive")
+    protected Button addToCart;
+
+    @FindBy(xpath = "//a[@title='Корзина']")
+    protected Button cart;
+
+    @Override
+    protected PepperoniPage openPage() {
+        driver.navigate().to(props.getKeyProperty("catalogueURL") + OnlinerEndpoints.pepperoniEndpoint);
+        logger.debug("Navigation to the url...");
+        return this;
+    }
+
+    @Override
+    protected PepperoniPage isPageOpened() {
+        try {
+            WaitUtils.waitForVisibility(price);
+        } catch (TimeoutException e) {
+            logger.debug("Element isn't found...");
```

Comment:
What element? 
Here we are waiting for the element to check whether the page is opened so it would make sense to specify that as well

Reply:
fixed